charms/istio-pilot/config.yaml

```diff
@@ -1,4 +1,12 @@
 options:
+  cni-bin-dir:
+    type: string
+    default: ''
+    description: Path to CNI binaries, e.g. /opt/cni/bin. This configuration option will only work when first deploying istio.
+  cni-conf-dir:
+    type: string
+    default: ''
+    description: Path to conflist files describing the CNI configuration, e.g. /etc/cni/net.d. This configuration option will only work when first deploying istio.
```

Comment:
Can we add a link to something that describes how to get these, or example usage?  If you intend on adding a readme or something else later, that works too

Reply:
I think both information in the README and a link here are useful. Will update this file and I will add info in the README (see https://github.com/canonical/istio-operators/issues/366).